src/decisionengine/framework/engine/tests/fixtures.py

```diff
@@ -52,9 +67,15 @@ def de_client_run_cli(self, *args):
         Run the DE Client CLI with these args
         The DE Server host/port are automatically set for you
         '''
-        return de_client.main(["--host", self.server_address[0],
-                               "--port", str(self.server_address[1]),
-                               *args])
+        return de_client.main(
```

Comment:
Personally, I find more clear the previous form. I do understand this is more consistent. Just a note, OK to keep.